app/src/main/res/values/strings.xml

```diff
@@ -103,12 +103,16 @@ username</string>
     <string name="label_block">Block</string>
     <string name="label_leave_group">Leave Group</string>
 
-
     <!-- errors -->
     <string name="error_unknown_title">Unknown error</string>
     <string name="error_unknown_message">Please try again in a moment</string>
     <string name="error_server_miscommunication_title">Server error</string>
     <string name="error_server_miscommunication_message">Please try again in a moment</string>
     <string name="error_no_network_title">No network error</string>
     <string name="error_no_network_message">Please check your Internet connection</string>
+
+    <!-- Animation label -->
+    <string name="animation_label_messagecomposeinput_state_transistion">MessageComposeInputState transition</string>
+    <string name="animation_label_button_rotation_degree_transistion">Collapse button rotation degree transition</string>
+
```

Comment:
Are these being used?

Also, `<string translatable="false" name="..."` might be better for our localisation tools and to get Android build to stop complaining about it.